Add `send_streaming_get` method to handle large files

dmalinovsky opened the pull request:
Also updated required libraries versions.

Commit:
Update to BeatifulSoup4 latest version

Commit:
Add send_streaming_get method to allow get large files

Commit:
Set a better chunk_size default (instead of 1 byte)

scaryclam commented:
Hi. Apologies for not replying earlier.

Do you have any tests available for you changes?

dmalinovsky commented:
No, I have none.  I don't have high hopes for this code, it's used in production, though, so it can be useful for someone else.

firech13f committed:
Metadata Ingestion for IdP! 

This commit adds a file to this repository, to ingest metadata, making it officially part of the EverHealth IdP. 🚀

🔧 What's inside?
✅ Metadata ingestion file 
✅ Integration with EverHealth IdP 
✅ One step closer to a unified catalog!
